src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -107,6 +107,15 @@ def check_cluster_connection(self, url, payload, username, ssl_verify, ssl_certi
             if 'token' not in content:
                 raise DashboardException(msg=content['detail'], code='invalid_credentials',
                                          component='multi-cluster')
+            
+            multi_cluster_content = self._proxy('POST', url, 'api/multi-cluster/auth', payload=payload,
+                                  verify=ssl_verify, cert=ssl_certificate)
+            if 'status' in multi_cluster_content and multi_cluster_content['status'] == '404 Not Found':
+                raise DashboardException(msg='The ceph cluster you are attempting to connect \
+                                         to lacks the multi-cluster feature. Please ensure that \
+                                         the cluster you are connecting to is upgraded to \
+                                         IBM Ceph 7.1 to enable the multi-cluster functionality.',
+                                         code='invalid_version', component='multi-cluster')
```

Comment:
The Ceph cluster you are attempting to connect \
doesn't  support Multi cluster feature. Please ensure that \
the cluster you are connecting to is upgraded to \
{Version We get from api of hub cluster} version to enable the Multi cluster feature.'